lite/valor_lite/detection/computation.py

```diff
@@ -452,13 +452,13 @@ def compute_detailed_counts(
 
     Index 0 - True Positive Count
     ... Datum ID Examples
-    Index n_samples + 1 - False Positive Misclassification Count
+    Index 2 * n_samples + 1 - False Positive Misclassification Count
```

Comment:
can you add a quick test (in one of the existing test functions) that shows what happens as you adjust n_sample (e.g., having `n_sample` be greater than the number of actual observations)?

Reply:
This is already tested in `test_compute_detailed_counts` look for examples that are `-1`.